extras/configure_gluster_collectd.yml

```diff
@@ -0,0 +1,39 @@
+---
+- name: Install and configure gluster collectd package
+  hosts: all
+  vars:
+    copr_url: "https://copr-be.cloud.fedoraproject.org/results/redara"
+    repo: "gluster-collectd/fedora-27-x86_64/00774074-gluster-collectd"
```

Comment:
url as 1 variable. there should be a way to provide long url in ansible.

Reply:
@sabose there is no limitation. I think it has been broken just for the maintainability I think.